src/content/en/updates/posts/2015/04/autocapitalize.markdown

```diff
@@ -100,39 +95,43 @@ Capitalization_. This is a change from the default behaviour.
 
 *  `<textarea autocapitalize="terry"></textarea>` would be a text area with _Sentences Capitalization_  
 *  `<textarea></textarea>` would be a text area with _Sentence Capitalization_.
-*  `<textarea autocapitalize=none></textarea>` would be a text area with _No Capitalization_.
+*  `<textarea autocapitalize="none"></textarea>` would be a text area with _No Capitalization_.
+
+For `HTMLFormElement` we have decided not to implement the attribute, because we've
+found that it is rarely used on pages today, and when it is used, it is mostly
+used to disable autocapitalization on the form entirely:
+
+`<form autocapitalize=off><input></form>`
 
-For _HTMLFor_mElement we have decided not to implement the attribute, we have
-found that it is rarely used on pages today, and when it is used it is mostly
-used to disable it on the form entirely `<form autocapitalize=off><input></form>`  which is odd, as the default state for `HTMLInputElement` is No Capitalization.
+The above is odd, as the default state for `HTMLInputElement` is _No Capitalization_.
 
 ## Why are you using this over `inputmode`?
 
-inputmode is meant to do solve the same type of problem, among other things.
-However it has been lacking Browser implementations &mdash; to the best of our
+`inputmode` is meant to solve the same type of problem, among other things.
+However, it has been lacking browser implementations &mdash; to the best of our
 knowledge, only Firefox OS has an implementation and it is prefixed
 (x-inputmode) &mdash; but it also has very little usage on the web. On the other hand,
-`autocapitalize` is used across Millions of pages on hundred of thousands of
+`autocapitalize` is used across millions of pages on hundred of thousands of
 websites already.
 
 ## When should I use this?
 
-This isn't an exhaustive list of when you should use \`autocapitalize\`, however
-there a number of places where helping the user enter text helps a great deal.
+This isn't an exhaustive list of when you should use `autocapitalize`; however
+there a number of places where aiding the user enter text helps a great deal:
 
```

Comment:
should be "aiding the user who is entering text" 

I'm pointing out that there's a faulty verb after your change.